src/components-examples/material/form-field/form-field-custom-control/form-field-custom-control-example.ts

```diff
@@ -115,6 +164,21 @@ export class MyTelInput implements ControlValueAccessor, MatFormFieldControl<MyT
     }
   }
 
+  autoFocusNext(
```

Comment:
nit: the name, args, return type can be all one line

Reply:
resolved